src/routes/establishment/routes.js

```diff
@@ -0,0 +1,12 @@
+import express from 'express';
+import create from './create';
+import destroy from './destroy';
+import get from './get';
+import getList from './getList';
+
+export default express
```

Comment:
Cool way of exporting the router here! I've typically just router.<verb>() ... repeatedly then exported the router after but this works nicely!